packages/edge-login-ui-rn/src/common/reducers/TermsAndConditinsReducer.js

```diff
@@ -1,11 +1,8 @@
 import * as Constants from '../../common/constants'
-
-const titleOne =
-  'I understand that my funds are held securely on this device, not by Edge'
-const titleTwo =
-  'I understand that if I lose this device or uninstall the app, my digital assets can only be recovered with my username and password'
-const titleThree =
-  'I understand that if I lose my username and password, Edge will not be able to recover my account, unless I setup password recovery'
+import s from '../../common/locales/strings.js'
+const titleOne = s.strings.terms_one
+const titleTwo = s.strings.terms_two
+const titleThree = s.strings.terms_three
```

Comment:
We are trying to avoid extra string constants like this. You could just put the `s.strings.terms_one` bits in the table right below.

Reply:
yup good catch , will change